src/components/panel/panel.js

```diff
@@ -1529,8 +1634,11 @@ MdPanelRef.prototype._configureTrapFocus = function() {
 
 
 /**
+ * @description
  * Animate the panel opening.
- * @returns {!angular.$q.Promise}
+ *
+ * @returns {!angular.$q.Promise} A promise that is resolved when the panel has
+ * animated open.
```

Comment:
continuation should be indented. This is the same as for params. (Indent 4 spaces from the @returns)